modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -1423,6 +1426,109 @@ neo4j_home$ --nodes persons.csv --nodes games.csv --id-type string
+
+[NOTE]
+====
+This feature is supported only by `block` format.
+====
+
+=== Set an explicit action for each row
+
+You can set an explicit action for each row in the CSV file by using the `:ACTION` keyword in the header file.
+If no action is specified, the import tool works as in full import mode, creating new data.
+
+The following actions are supported:
+
+* `empty` = `CREATE` (default)
+* `C`, `CREATE` - Creates new nodes and relationships, with or without properties, as well as labels.
+* `U`, `UPDATE` - Updates existing nodes, relationships, labels, and properties.
+* `D`, `DELETE` - Deletes existing nodes or relationships.
+Deleting a node also deletes its relationships (`DETACH DELETE`).
+
+
+.Using actions in CSV files to update nodes
+[source, cypher, role="nocopy"]
+----
+:ACTION,personId:ID,name,:LABEL
+CREATE,1,"Keanu Reeves",Actor
+UPDATE,2,"Laurence Fishburne",Actor
+DELETE,4
+----
+
+Nodes are identified by their unique property value for the key/label combination that the header specifies.
+
+.Using actions in CSV files to update relationships
+[source, cypher, role="nocopy"]
+----
+:ACTION,:START_ID,:END_ID,:TYPE,role
+CREATE,1,2,ACTED_IN,"Neo"
+UPDATE,1,2,ACTED_IN,"Morpheus"
+DELETE,1,3,ACTED_IN
+----
+
+Relationships are identified non-uniquely by their start and end node IDs, and their type.
+
+To further narrow down selection you can tag a property column as an identifier to help out in selecting relationships uniquely (or at least more uniquely).
+
+.Using actions in CSV files to update relationships with identifier properties
+[source, cypher, role="nocopy"]
+----
+:ACTION,:START_ID,:TYPE,:END_ID,p1{identifier:true},name,p4
+U,1,KNOWS,2,abc,"Keanu Reeves","Hello Morpheus"
+U,1,KNOWS,2,def,"Laurence Fishburne","Hello Neo"
+----
+
+The data in the `p1` column for these relationships acts "more uniquely" selecting relationships, if multiple of `1,KNOWS,2` exist.
+Identifier properties will match the selected relationships and as such not be set on the relationships (they already have it).
+
+=== Update existing labels
+
+You can add or remove one or more labels from an existing node by prepending the clause `LABEL` in the header with a `+` (default) or `-`:
+
+* `:+LABEL` - Add one or more labels to an existing node.
+* `:-LABEL` - Remove one or more labels (if they exist) from an existing node.
+
+Multiple labels are separated by a semicolon (default).
+
+For example, a file could have the following format:
+
+[source, csv]
+----
+:ID,:+LABEL,:-LABEL,name,age
+person1,Actor,Person,"Keanu Reeves",55
+person2,Actor;Director,Person,"Laurence Fishburne",60
+person3,Actor,Person,"Carrie-Anne Moss",53
+----
+
+In this case, all labels in the second column are added and all the labels in the third column are removed (if they exist).
+
+=== Remove existing properties
+
+You can remove properties from existing nodes or relationships by specifying them after the keyword `:-PROPERTY` in the header file.
```

Comment:
Suggestion: "You can remove properties from existing nodes or relationships by adding a `:-PROPERTY` column in the header. In the contents of this field you can add zero or more property names to remove from the entity".